SAMv1.tex

```diff
@@ -308,10 +308,10 @@ \subsection{The header section}
   & {\tt LB} & Library.\\\cline{2-3}
   & {\tt PG} & Programs used for processing the read group.\\\cline{2-3}
   & {\tt PI} & Predicted median insert size.\\\cline{2-3}
-  & {\tt PL} & Platform/technology used to produce the reads. \emph{Valid values}:
-  {\tt CAPILLARY}, {\tt LS454}, {\tt ILLUMINA}, {\tt SOLID}, {\tt HELICOS}, {\tt IONTORRENT}, {\tt ONT}, and {\tt PACBIO}.\\\cline{2-3}
-  & {\tt PM} & Platform model. Free-form text providing further details of the
-  platform/technology used.\\\cline{2-3}
+  & {\tt PL} & Platform/technology used to produce the reads. \emph{Valid values are}:
+  {\tt BGI}, {\tt CAPILLARY}, {\tt HELICOS}, {\tt ILLUMINA}, {\tt IONTORRENT}, {\tt LS454}, {\tt ONT}, {\tt PACBIO}, {\tt SOLID}.
+  Tag should be omitted when the technology is not in this list, (though the {\tt PM} tag may still be present in this case) or is unknown.\\\cline{2-3}
```

Comment:
The comma is misplaced.

Personally I prefer just “Omitted when the […]” because it side-steps concerns about _must_ vs _should_ etc — it just _is_. But if you really want to put something more in front, it would gain from “This tag” or so; and “field” is better than “tag” because it clearly encompasses the whole `TAG:VALUE` item.

```suggestion
  This field should be omitted when the technology is not in this list (though the {\tt PM} field may still be present in this case) or is unknown.\\\cline{2-3}
```